src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAXIMUM_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        Random random = new Random();
```

Comment:
You don't need to create new `Random` instance every time you call `getRandomBall`, introduce it as field variable.